### DISCUSSION TITLE
[#1040946] Change brush size with mouse wheel

### DISCUSSION THREAD

**Discussion by TORD31** [Mojang/minecraft-editor#73] [Feedback]
For example when you build a mountain you may often need to change the brush size for the fine tuning. Therefore it would be helpful to easily change the brush size with your mouse wheel because it would make this process a little bit faster.

**Comment by minecraftchloe** [comment]
This would be awesome!! Meta requirement appears to be to allow key/mousebindings on individual panel elements (eg. the brush size slider).
